### ISSUE TITLE
I'm catching an exception with attachment_url

### ISSUE THREAD

**Issue by BornALegend** [elabs/refile#124]
I created a helper method to support my messages controller, and I can't seem to figure out why I'm getting an error while passing the attachment_url and the user object

Here is the code for the method.
module MessagesHelper
<option value='#{user.id}'
data-img-src='#{attachment_url(@user, user.email, :profile_avatar,
:fill, 50, 50)}'>#{user.username}</option>"

How to resolve this issue when the image isn't available? if nil?

**Comment by jnicklas** [comment]
This is incorrect:

``` ruby
attachment_url(@user, user.email, :profile_avatar, :fill, 50, 50)
```

What is `user.email` doing in there? You want something like this:

``` ruby
attachment_url(@user, :profile_avatar, :fill, 50, 50)
```

**Comment by BornALegend** [comment]
I did what you said correctly, but I need to create a helper method that passes multiple objects into the method body. I started this process with gravatar, and I'm still learning how to utilize refiles strengths

My original helper method for gravatar is below

def gravatar_for(user, size= 30, title = user.username)
image_tag  gravatar_image_url(user.email, size: size), title: title, class: 'img-rounded'
end

I need the alternative for refile. The method that I started the issues thread with will work if I complete the sequence.
